### DISCUSSION TITLE
How can I add translate based on tilting to the element?

### DISCUSSION THREAD

**Discussion by zalishchuk** [mkosir/react-parallax-tilt#48] [Ideas]
Hey! Can you explain, is it possible to not only tilt the element but also change its `translateX` and `translateY` with the same maxed values as rotate to get that fancy parallax effect when the card is tilting and simultaneously moves slightly closer to the cursor? The problem is that I can not append or reassign the `transform` CSS property because `react-parallax-tilt` reassigns it every time.

**Comment by mkosir** [comment]
Hey @zalishchuk I really like the idea you proposed 🔥 
But currently this is not possible to do, since as you mentioned `transform` property gets overwritten. The only way would be to extend the API of the library to also accept `translate` property.
